jbake-core/src/main/java/org/jbake/parser/texy/TexyServiceEngine.java

```diff
@@ -0,0 +1,44 @@
+package org.jbake.parser.texy;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import org.jbake.parser.MarkupEngine;
+import org.jbake.parser.ParserContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Renders documents in the Texy syntax.
+ *
+ * @author Ondřej Žižka
+ */
+public class TexyServiceEngine extends MarkupEngine
+{
+
+    private static final Logger logger = LoggerFactory.getLogger(TexyServiceEngine.class);
+
+    @Override
+    public void processBody(final ParserContext context) {
+        String documentBody = context.getBody();
+
+        try (InputStream stream = new ByteArrayInputStream(documentBody.getBytes(StandardCharsets.UTF_8))){
+            TexyRestService texyService = new TexyRestService(new URL("http://localhost:8022/TexyService.rest.php"));
```

Comment:
This seems like a candidate for another configuration option. Maybe a check if the service is available and handle the error?

Reply:
Config - sure, I am looking at how the config is propagated around.
In the end it will have around 8 config entries, which fine-tune Texy's behavior.

Check for availability - yes. I would basically just throw an exception and it should be up to JBake to tell the user that "couldn't convert document XY" + the reason.

Reply:
you can find the configuration in the ParserContext class.